gearcatalog/frontend/src/components/gear/GearList.tsx

```diff
@@ -51,7 +51,7 @@ const GearList = ({ deleteGear, getGear, gearList, token}: GearListProps) => {
                 <strong>{gear.name}</strong>
               </div>
               <div className="card-body">
-                <p className="card-text">Brand: {gear.brand} </p>
+                <p className="card-text">Quantity: {gear.qty}</p>
```

Comment:
What is the reason to replace brand with quantity?

Reply:
quantity seems more useful piece of info. When I had both it inserted a new line and it looked bad. So I went with qty.